frontend/index.html

```diff
@@ -1,6 +1,18 @@
 <!doctype html>
 <html lang="ko">
   <head>
+    <link
+      rel="preconnect"
+      href="https://cdn.jsdelivr.net"
+      crossorigin="anonymous"
+    />
+    <link
+      rel="preload"
+      as="style"
+      href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/variable/pretendardvariable-dynamic-subset.min.css"
```

Comment:
아르 덕분에 가변체에 대해 알았네요!
직관적으로 모든 폰트의 굵기를 담고 있는 서체라면 오히려 더 용량이 무거운 건 아닌가 하는 위문이 들었는데, 개별 파일보단 약간 무거울 수 있지만 다른 굵기의 파일을 여러 개 가져오는 것 보다 훨씬 가볍군요👀

100단위의 정해진 수가 아닌 그 사이의 수를 마음대로 조절할 수 있다는 점도 재밌는 포인트네요.ㅎㅎ

> Traditionally, all possible weights and styles have been separated out into different font files, whereas variable fonts combine all of those variations into one. Because of this, overall file size is greatly reduced compared to loading multiple individual font files—and that’s a key consideration for web typography.